evap/staff/views.py

```diff
@@ -162,8 +160,14 @@ def semester_evaluation_operation(request, semester_id):
 
     if request.method == 'POST':
         template = None
+        template_contributor = None
+        template_participant = None
         if request.POST.get('send_email') == 'on':
             template = EmailTemplate(subject=request.POST['email_subject'], body=request.POST['email_body'])
```

Comment:
['email_subject'] results in a `MultiValueDictKeyError`. did you even test this? :)

Reply:
That's a good point, forgot to set to WIP. Yes, I haven't tested yet, so far it's been code only. Sorry for the confusion.